content/about/faq.md

```diff
@@ -1,23 +1,77 @@
+### What are the benefits of 11ty?
 
-**Who makes up the Quire community?**
+For editors and content creators:
 
-**Why Open Source?**
+- The current functionality and design of Quire websites, PDFs, and ebooks will remain the same
+- There will be minor differences within the Markdown files, such as a new format for shortcodes:  `{% figure id="x" %}` 
+- Core configuration variables will be restructured to be more user-friendly
+- Key project files will become more accessible and simplify navigation
+- Development support will be easier to access due to a more familiar and approachable technology stack
 
-**How can I participate?**
+For developers:
+
+- Quire will be entirely JavaScript
+- Knowledge of Go will not be necessary to take full advantage of Quire's features
+- It will be easier to create new templates that work with publication data or use data from remote sources
+- Built-in ability to mix templating languages; Quire will primarily use Liquid for simple markup and JavaScript where data manipulation is required
+- Templates and code will be refactored and implemented as functional, stateless, 11ty shortcode components using JavaScript
+- Interactive elements, such as the image lightbox, will be reimplemented using Web Components
+- To enable deeper integration with the IIIF APIs, Leaflet.js will be replaced with the CanvasPanel web components
+- The themes folder will be eliminated so customizing projects will be based on the 11ty framework structure for includes and plugins
+- Improved developer support and contribution opportunities
+
+### What is JavaScript?
+
+JavaScript is a popular text-based programming language that turns static web pages into dynamic, interactive sites, going a step beyond the capabilities of HTML and CSS. JavaScript can be found everywhere on the web, from refreshing a Twitter feed to your Amazon search bar or embedding Youtube videos. It is also widely used in the art, cultural heritage, and academic sectors. Learn more about JavaScript on the [Mozilla developer website](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/What_is_JavaScript).
+
+### What is Liquid?
+
+Liquid is an open-source templating language written in [Ruby](https://www.ruby-lang.org/en/). It was created by the e-commerce company [Shopify](https://www.shopify.com/) in 2006. It is utilized in all Shopify themes to load dynamic content on website storefronts. Learn more about Liquid on the [Shopify.dev website](https://shopify.dev/api/liquid).
+
+### What is Vite?
+
+Vite is a JavaScript bundler and development web server. It will replace Webpack as a tool to bundle scripts and styles for the web. Learn more on the [Vite website](https://vitejs.dev/).
+
+### What is CanvasPanel?
```

Comment:
The link you found about Canvas Panel is outdated. Let's use this one: https://iiif-canvas-panel.netlify.app/. Also note that they seem to have added a space to the name so it's "Canvas Panel" and not "CanvasPanel". Sorry I didn't catch it before.